test/queries.js

```diff
@@ -166,3 +281,9 @@ graphql(BigchainDBSchema, queryVoteList).then(result => {
 graphql(BigchainDBSchema, querySearch).then(result => {
     console.log(JSON.stringify(result, null, 2))
 })
+graphql(BigchainDBSchema, mutationTransaction).then(result => {
```

Comment:
I prepopulated database with prepopulate.py from python graphql. I change order so everything happens from queries.js without need of prepopulating from python-graphql